### ISSUE TITLE
vehicle lock/unlock button change.

### ISSUE THREAD

**Issue by ahsan009** [ESX-PUBLIC/esx_vehiclelock#24]
is it possible to change vehicle lock/unlock button from u to g?
if yes then how ?

**Comment by gaby** [comment]
Yes, change the number in this line https://github.com/ESX-PUBLIC/esx_vehiclelock/blob/master/client/main.lua#L83

303 is the letter U

You can find all the other keys here: https://docs.fivem.net/docs/game-references/controls/
